File: crates/fresh-editor/tests/fixtures/syntax_highlighting/hello.dart
// Dart syntax highlighting test
import 'dart:io';

String greet(String name) {
  return 'Hello, $name!';
}

class Config {
  final String version;
  final bool enabled;
  final int count;

  Config({
    required this.version,
    this.enabled = true,
    this.count = 42,
  });
}

void main() {
  final message = greet('World');
  print(message);

  final items = [1, 2, 3, 4, 5];
  for (final item in items) {
    print('Item: $item');
  }
}

File: crates/fresh-editor/tests/fixtures/syntax_highlighting/hello.jl
# Julia syntax highlighting test
module Hello

function greet(name::String)::String
    return "Hello, $name!"
end

struct Config
    version::String
    enabled::Bool
    count::Int64
end

function main()
    message = greet("World")
    println(message)

    items = [1, 2, 3, 4, 5]
    for item in items
        println("Item: $item")
    end
end

end # module
